docs/sphinx/methods/dsrg.rst

```diff
@@ -16,10 +16,18 @@ Driven Similarity Renormalization Group
   Depending on the features used, the user is encouraged to cite the corresponding articles listed :ref:`here <dsrg_ref>`.
 
 .. caution::
-  The current implementation does not employ spin-adapted equations and it does not work for even spin multiplicities (doublet, quartets, etc.).
-  For odd multiplicities, we assume low-spin configurations (by default, no need to specify in the input file).
-  For those in desperate need to perform computations on doublets, an alternative way is to add a hydrogen atom far away from the system and perform a singlet computation.
-  Spin adaptation is on the TODO list.
+  The examples used in this manual are written based on the spin-integrated code
+  that works only for singlet molecules and those with odd multiplicities.
+  To invoke the spin-adated implementation (that works for general multiplicities),
+  the user needs to specify the following keywords:
+  ::
+
+     correlation_solver  sa-mrdsrg  # spin-adapted DSRG computation
+     spin_avg_density    true       # must use spin-averaged densities
```

Comment:
Could we get rid of the variable `spin_avg_density`? At least we should have logic to enforce that when `sa-mrdsrg` is set then `spin_avg_density = True`.

Reply:
OK, let me work on it.